README.md

```diff
@@ -134,6 +134,50 @@ until the constructor for `config.Provider` can be fully satisfied.
 If resolution is not possible, for instance one of the required dependencies has
 does not have a constructor and doesn't appear in the graph, an error will be returned.
 
+## Invoke
+
+`Invoke` API executes the provided function and stores return results in the dig container.
+
+`Invoke` looks through the dig graph and resolves all the constructor parameters for execution.
+The return results, except the `error` object is inserted into the graph for further use.
+To utilize `Invoke`, function needs following form -
+
+1. function arguments are pointers, maps, slice or arrays
+2. function arguments must be provided to dig
```

Comment:
Saying "must" here but not in #1, for example, makes it seem that the function arguments can be something other than pointers, maps, slices, or arrays. Every list item _must_ be followed, right? If so, change "must be" to "are".